packages/libs/client/README.md

```diff
@@ -245,9 +247,41 @@ const signedTransaction = signWithChainweaver(unsignedTransaction)
 There's several steps to setup a WalletConnect connections and sign with
 WalletConnect.
 
-1. Setting up the connection using [`ClientContextProvider.tsx`][30]
+1. Setting up the connection using [`ClientContextProvider.tsx`][32]
 2. Use `signWithWalletConnect` to request a signature from the wallet
-   (`Transaction.tsx`)\[[https://github.com/kadena-io/wallet-connect-example/blob/2efc34296f845aea75f37ab401a5c49081f75b47/src/components/Transaction.tsx#L104][31]]
+   (`Transaction.tsx`)\[[https://github.com/kadena-io/wallet-connect-example/blob/2efc34296f845aea75f37ab401a5c49081f75b47/src/components/Transaction.tsx#L104][33]]
+
+## Verifier
+
+Kadena supports `verifier` as another way of gaining authority in Pact, as well
+as the normal signature flow. In this way, a `verifier`, `proof`, and the
+`capability list` will be sent to the blockchain, and if the proof satisfies the
+verifier function, the capabilities will be granted. It's useful, for example,
+for ZK (Zero Knowledge Proof) or bridging between networks.
```

Comment:
@edmundnoble 
Is this explanation right?

Reply:
This is good yes.